.github/workflows/tests.yml

```diff
@@ -97,6 +97,26 @@ jobs:
           directory: ./coverage
           flags: front,unit_front
 
+  build:
+    name: 'build (node: ${{ matrix.node }})'
+    needs: [lint, unit_front]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14, 16, 18]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ matrix.node }}-${{ hashFiles('**/yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+      - name: Build
+        run: yarn build
```

Comment:
Should we only try the admin panel / helper plugin builds here ?

Reply:
Any reason you'd choose to omit DTS? At the moment the only other build iirc.

Suppose we're gonna include more build steps that are just TSC transpiration in the future?

Reply:
Yeah just because It's already built for both lint & back unit tests